sormas-api/src/main/java/de/symeda/sormas/api/user/UserRole.java

```diff
@@ -61,7 +61,8 @@
 	IMPORT_USER(false, false, false, false, JurisdictionLevel.NONE),
 	REST_EXTERNAL_VISITS_USER(false, false, false, false, JurisdictionLevel.NONE),
 	REST_USER(false, false, false, false, JurisdictionLevel.NONE),
-	SORMAS_TO_SORMAS_CLIENT(false, false, false, false, JurisdictionLevel.NONE);
+	SORMAS_TO_SORMAS_CLIENT(false, false, false, false, JurisdictionLevel.NONE),
+	BAG_USER(false, false, false, false, JurisdictionLevel.NONE);
```

Comment:
Please add the _BAG_USER constant bellow to be used in the @RolesAllowed annotation
Also add it to the `addAssignableRoles` method